src/ReviewService.Abstractions/ReviewService.Abstractions.csproj

```diff
@@ -11,5 +11,6 @@
 		<PackageProjectUrl>https://github.com/nventive/ReviewService</PackageProjectUrl>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
 		<PackageTags>review;maui;winui;ios;android</PackageTags>
+		<RootNamespace>ReviewService</RootNamespace>
```

Comment:
Obviously depending on my other comment, from my understanding, shouldn't the root namespace be `ReviewService.Abstraction` if every file on this package is that namespace